### ISSUE TITLE
write healpix FITS files

### ISSUE THREAD

**Issue by bhazelton** [RadioAstronomySoftwareGroup/pyradiosky#78]
(from notes in the code that I removed)

**Comment by mkolopanis** [comment]
Do we have copies of the specifications we want? Fits and healpix go together fairly well, the CMB community has been doing this for a while and should not require a ton of work.

**Comment by bhazelton** [comment]
We need to talk about this. Standard HEALPix FITS files do not support a frequency axis. We need to decide whether to make a new non-standard file type or make different HDUs or files for different frequencies.

**Comment by mkolopanis** [comment]
Yeah we could end up with a large number of HDUs if all the frequencies are put into one file. I've seen things similar to this before, but usually integrated bandpowers which result in 5ish HDUs (e.g. WMAP cmb maps).

**Comment by jpober** [comment]
This may be particularly useful for Ruby's catalog.  We'll probably need a read method also.

**Comment by jpober** [comment]
@rlbyrne Have your plans for releasing your catalog/map become any clearer?  Do you still plan on a skyh5 and a more standard HEALPix release?  For the latter, how do you plan to create that file?

**Comment by rlbyrne** [comment]
@jpober Yes, I'm planning to release the data in both formats. For the Healpix fits format, I have code separate from pyradiosky. That could get integrated into pyradiosky at some point, but Bryna's concerns about the frequency axis still stand.

**Comment by jpober** [comment]
Ok, I think we will want to consider that code for integration into pyradiosky (at some point).  I can still see at being nice functionality for single frequency maps.
